pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -283,6 +290,10 @@ type ServicePlan struct {
 	// ServiceInstanceCredentialCreateParameterSchema is the schema for the parameters that
 	// may be supplied binding to an ServiceInstance on this plan.
 	ServiceInstanceCredentialCreateParameterSchema *runtime.RawExtension `json:"serviceInstanceCredentialCreateParameterSchema,omitempty"`
+
+	// ServiceClassRef is a reference to the service class that
+	// owns this plan.
+	ServiceClassRef v1.LocalObjectReference `json:"serviceClassRef"`
```

Comment:
Not necessary for this PR unless you really want to, but since this property isn't set by the user I think it should probably live in the Status instead.  We may need to check all of our resources for this type of thing.

Reply:
Service class and plan do not have the status/spec split as data only objects.

Reply:
ah forgot about that

Reply:
Let's keep the model as similar as possible for now.  I think it would be a good idea to have spec and status for these objects which we can talk about later.

Reply:
Even if we introduce spec and status here, I don't think it is a good idea to put `serviceClassRef` into `status`.
It was widely recognized as an overlook in the Kubernetes API v1 design that user-provided and autogenerated fields both live in spec, but it works this way until v2 and we can't and shouldn't mask this issue by putting non-status data into status.